src/clusterfuzz/_internal/tests/core/metrics/fuzzer_stats_test.py

```diff
@@ -252,10 +252,18 @@ def test_fuzz_task_upload_job_run_stats(self):
     ]
     groups[0].is_new.return_value = False
     groups[1].is_new.return_value = True
-
+    crashes = fuzz_task.convert_groups_to_crashes(groups)
+    job_run_crashes = []
+    for crash in crashes:
+      job_run_crashes.append({
+          'is_new': crash.is_new,
+          'count': crash.count,
+          'crash_type': crash.crash_type,
+          'crash_state': crash.crash_state,
+          'security_flag': crash.security_flag,
+      })
```

Comment:
This is the third time this code is repeated, maybe extract it to a helper somewhere, maybe in `uworker_io`?